scripts/time_plot.py

```diff
@@ -1497,6 +1497,69 @@ def handle_websock_event(handlerkey,*args):
                 for thishandler in websockrequest_username.keys():
                     if (websockrequest_username[thishandler]==username):
                         send_websock_cmd('ApplyViewLayout('+'["'+'","'.join([fig['figtype'] for fig in html_viewsettings[username]])+'"]'+','+str(html_layoutsettings[username]['ncols'])+')',thishandler)
+        elif (args[0].startswith('holo')):#accepts 'holo', 'holo 3' where 3 is m003 as explicit reference antenna, else first track antenna is chosen as reference antenna
+            logger.info(repr(args))
+            antnumbers=[int(antnumberstr[1:]) for antnumberstr in telstate_antenna_mask]#determine all available inputs
+            if (len(antnumbers)==0):
+                send_websock_cmd('logconsole("No antenna inputs found or specified",true,true,true)',handlerkey)
+            else:
+                cbid=str(telstate['sdp_capture_block_id'])
+                obs_params_key=telstate.join(cbid, 'obs_params')
+                obs_params=telstate.get(obs_params_key, {})
+                scan_ants=[]
+                track_ants=[]
+                if 'scan_ants' in obs_params:
+                    scan_ants=obs_params['scan_ants'].split(',')
+                if 'track_ants' in obs_params:
+                    track_ants=obs_params['track_ants'].split(',')
+                if 'scan_ants_always' in obs_params:
+                    scan_ants_always=obs_params['scan_ants_always'].split(',')
+                    newtrackants=[]
+                    for ant in track_ants:
+                        if ant in scan_ants_always:
+                            scan_ants.append(ant)
+                        else:
+                            newtrackants.append(ant)
+                    track_ants=newtrackants
+                    scan_ants=sorted(scan_ants)
+                if (len(args)==1 or args[1]==''):
+                    if len(track_ants):
+                        refantnumber=int(track_ants[0][1:])
+                    else:
+                        refantnumber=antnumbers[0]
+                else:#use supplied refant
+                    refantnumberlist=parse_antennarange(','.join(args[1:]))
```

Comment:
Use `ref_ant_number_list` for consistency.

Reply:
I think there are too many of these inconsistencies around, e.g. newtrackants, refantnumber, refantnumberlist, cbid, antnumbers, and also in other nearby blocks of code. It is probably better to just leave it unless a concerted effort is done to change everything throughout all the source code.

Reply:
One day we're going to make you actually fix all the awfulness that you keep putting off because all the surrounding code is awful :-)

Reply:
@ludwigschwardt spotted some deprecated usages of telstate and a few other points. I told him I should do a different pull request to address a bunch of random things like that (maybe next month). So it would be good if you compile a few suggestions for such a PR next month or so.